stdlib/public/PlatformDifferentiation/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+#===--- CMakeLists.txt - Platform differentiation library ----------------===#
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2019 - 2020 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+#===----------------------------------------------------------------------===#
+
+add_swift_target_library(swift_Darwin_Differentiation ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_SDK_OVERLAY
+  "${SWIFT_SOURCE_DIR}/stdlib/linker-support/magic-symbols-for-install-name.c"
+
+  GYB_SOURCES
+    TgmathDerivatives.swift.gyb
+
+  SWIFT_MODULE_DEPENDS_OSX Darwin _Differentiation
+  SWIFT_MODULE_DEPENDS_IOS Darwin _Differentiation 
+  SWIFT_MODULE_DEPENDS_TVOS Darwin _Differentiation
+  SWIFT_MODULE_DEPENDS_WATCHOS Darwin _Differentiation
+
+  SWIFT_COMPILE_FLAGS
+    ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS}
+  LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
+  TARGET_SDKS ALL_APPLE_PLATFORMS FREESTANDING
+  INSTALL_IN_COMPONENT sdk-overlay)
+
+
+add_swift_target_library(swift_Glibc_Differentiation ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_SDK_OVERLAY
+  GYB_SOURCES
+    TgmathDerivatives.swift.gyb
+
+  SWIFT_MODULE_DEPENDS_LINUX Glibc _Differentiation
+  SWIFT_MODULE_DEPENDS_FREEBSD Glibc _Differentiation
+  SWIFT_MODULE_DEPENDS_OPENBSD Glibc _Differentiation
+  SWIFT_MODULE_DEPENDS_CYGWIN Glibc _Differentiation
+  SWIFT_MODULE_DEPENDS_HAIKU Glibc _Differentiation
+
+  SWIFT_COMPILE_FLAGS
+    ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS}
+  LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
+  TARGET_SDKS ANDROID CYGWIN FREEBSD OPENBSD LINUX HAIKU
+  INSTALL_IN_COMPONENT sdk-overlay)
+
+add_swift_target_library(swift_CRT_Differentiation ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_SDK_OVERLAY
+  GYB_SOURCES
+    TgmathDerivatives.swift.gyb
+
+  SWIFT_MODULE_DEPENDS_WINDOWS CRT _Differentiation
+
+  SWIFT_COMPILE_FLAGS
+    ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS}
+  LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
+  TARGET_SDKS WINDOWS
+  INSTALL_IN_COMPONENT sdk-overlay)
```

Comment:
I don't see anything, either here or in the platform modules, that would create the `{Darwin,Glibc,CRT}.swiftcrossimport/_Differentiation.swiftoverlay` YAML file you'd need to make the cross-import overlay work correctly. Did I miss something?

Reply:
I wasn't familiar with how the internals of cross-import overlays work, and will to look into that, once I've resolved the build issues.

Reply:
You can use the draft Evolution proposal as documentation: https://gist.github.com/beccadax/8f35216340e829efa5e9c1fc090fda09